src/components/insurance/PolicyItem.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div class="aig-policy-item">
+    <div class="policy-list-item-img-container">
+      <img class="product-img" :src="policy.productImg">
+    </div>
+    <div class="policy-list-item-content">
+      <div class="policy-item-header">
+        <div class="text">
+          {{$t('insurance.policy.policyId')}}
+        </div>
+        <div class="id">
+          {{policy.id}}
+        </div>
+      </div>
+      <div class="policy-item-body">
+        <div class="policy-info-item">
+          <div class="title">
+            <p class="title">{{$t('insurance.policy.deviceId')}}</p>
+          </div>
+          <div class="content">
+            {{policy.deviceId}}
+          </div>
+        </div>
+        <div class="policy-info-item-date">
+          <div class="title">
+            <p class="title">{{$t('insurance.policy.startDate')}}</p>
+          </div>
+          <div class="content">
+            {{policy.startUtc ? policy.startUtc : 'N/A'}}
+          </div>
+        </div>
+        <div class="policy-info-item-date">
+          <div class="title">
+            <p class="title">{{$t('insurance.policy.endDate')}}</p>
+          </div>
+          <div class="content">
+            {{policy.endUtc ? policy.endUtc : 'N/A'}}
+          </div>
+        </div>
+        <div class="policy-info-item-date">
+          <div class="title">
+            <p class="title">{{$t('insurance.policy.status')}}</p>
+          </div>
+          <div class="content">
+            {{policy.status}}
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    policy: {
+      required: true
```

Comment:
I don't think we need to validate props, the component would break if the prop is not provided so I don't think we need console warning. I would suggest using one liner "props: ['policy']"